openapi/auth-server.yaml

```diff
@@ -500,6 +500,18 @@ components:
       required:
         - redirect
         - finish
+    interact-state:
```

Comment:
We don't want this spec to change. It's the official auth server spec. The interaction here is between the auth server and the client, not the auth server and the resource server. Grant lookup is auth server - resource server interaction. This is specified here: `packages/auth/src/openapi/id-provider.yaml`

Instead of adding this schema here, could you add it in `packages/auth/src/openapi/id-provider.yaml` and then not reference to it?